### PR TITLE
Add Web3Go to Data Analytics

### PR DESCRIPTION
**Pull request by Moehringen**
I have added one line for the Web3Go project, which is the data analytics platform on Polkadot.

### CODE REVIEW

File: docs/build/build-data.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ chain-agnostic data aggregation, indexing, and querying layer between blockchain
 SubQuery abstracts away blockchain-specific data idiosyncrasies using the 
 [SubQuery SDK](https://github.com/subquery/subql).
 
+### Web3Go
+
+[Web3GO](https://web3go.xyz/) Web3Go is a data analytics platform on Polkadot focused on building professional dashboards and providing the tools and API to help users easily visualize, curate, and share data. currently, Web3Go focuses on building the infrastructure for data from the Polkadot ecosystem.
```
Comment (salmad3) [inline docs/build/build-data.md]:
```suggestion
[Web3GO](https://web3go.xyz/) is a Polkadot-based open data analytics platform that provides infrastructure and tooling to help users visualize, curate, share, and analyze on-chain data.
```

File: docs/build/build-data.md [hunk 2]
```diff
@@ -28,6 +28,11 @@ chain-agnostic data aggregation, indexing, and querying layer between blockchain
 SubQuery abstracts away blockchain-specific data idiosyncrasies using the 
 [SubQuery SDK](https://github.com/subquery/subql).
 
+### Web3Go
+
+[Web3GO](https://web3go.xyz/) Web3Go is a data analytics platform on Polkadot focused on building professional dashboards and providing the tools and API to help users easily visualize, curate, and share data. currently, Web3Go focuses on building the infrastructure for data from the Polkadot ecosystem. 
+
```
Comment (salmad3) [inline docs/build/build-data.md]:
```suggestion
```

File: docs/build/build-data.md [hunk 3]
```diff
@@ -28,6 +28,11 @@ chain-agnostic data aggregation, indexing, and querying layer between blockchain
 SubQuery abstracts away blockchain-specific data idiosyncrasies using the 
 [SubQuery SDK](https://github.com/subquery/subql).
 
+### Web3Go
+
```
Comment (salmad3) [inline docs/build/build-data.md:32]: also needs formatting (prettify)

**Review by salmad3 — CHANGES_REQUESTED** [review]
Thanks @Moehringen, left some comments

**Review by Moehringen — COMMENTED** [review]
thanks!

File: docs/build/build-data.md [hunk 4]
```diff
@@ -28,6 +28,10 @@ chain-agnostic data aggregation, indexing, and querying layer between blockchain
...
+[Web3GO](https://web3go.xyz/) is a Polkadot-based open data analytics platform that provides infrastructure and tooling to help users visualize, curate, share, and analyze on-chain data.
```
Comment (salmad3) [inline docs/build/build-data.md]:
```suggestion
[Web3GO](https://web3go.xyz/) is a Polkadot-based open data analytics platform that provides 
infrastructure and tooling to help users visualize, curate, share, and analyze on-chain data.
```

**Pull request merged**